apps/mobile/src/app/(home)/mpc-wallets.tsx

```diff
@@ -0,0 +1,110 @@
+import { useRef, useState } from 'react';
+import { ScrollView } from 'react-native-gesture-handler';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import { NotifyUserSheet, OptionData } from '@/components/sheets/notify-user-sheet.layout';
+import { Trans, t } from '@lingui/macro';
+import { useTheme } from '@shopify/restyle';
+
+import {
+  Box,
+  Cell,
+  LogoMPCBitgo,
+  LogoMPCCapsule,
+  LogoMPCCopper,
+  LogoMPCFireblocks,
+  LogoMPCFordefi,
+  LogoMPCPortal,
+  LogoMPCPrivy,
+  LogoMPCQredo,
+  QuestionCircleIcon,
+  SheetRef,
+  Text,
+  Theme,
+  TouchableOpacity,
+} from '@leather.io/ui/native';
+
+function getUnavailableFeatures() {
+  return {
+    bitgo: { title: t`BitGo`, Icon: LogoMPCBitgo },
+    capsule: { title: t`Capsule`, Icon: LogoMPCCapsule },
+    copper: { title: t`Copper`, Icon: LogoMPCCopper },
+    fireblocks: { title: t`Fireblocks`, Icon: LogoMPCFireblocks },
+    foredefi: { title: t`Foredefi`, Icon: LogoMPCFordefi },
+    portal: { title: t`Portal`, Icon: LogoMPCPortal },
+    privy: { title: t`Privy`, Icon: LogoMPCPrivy },
+    qredo: { title: t`Qredo`, Icon: LogoMPCQredo },
+  };
+}
+
+export default function MPCWalletsScreen() {
```

Comment:
```suggestion
export default function MpcWalletsScreen() {
```

elsewhere we treat acronyms like words when usig PascalCase